tools/tool_settings.h

```diff
@@ -1114,13 +1115,44 @@ GetVulkanReplayOptions(const gfxrecon::util::ArgumentParser&           arg_parse
         replay_options.preload_measurement_range = true;
     }
 
-    replay_options.dump_resources              = arg_parser.GetArgumentValue(kDumpResourcesArgument);
+    const std::string& dump_resources = arg_parser.GetArgumentValue(kDumpResourcesArgument);
+    if (!dump_resources.empty() && dump_resources.find_first_not_of("0123456789,") == std::string::npos)
+    {
+        // If the option parameter does not split into three comma separated numbers, consider
+        // it a Vulkan option and ignore it. If it does split into three comma separated numbers,
+        // the arg is for dx12 and should have already been validated in the Vulkan option parsing.
+        // In that case, we simply extract and save the values here.
```

Comment:
This comment is incorrect - an arg of 3 numbers separated by commas can now by used for Vulkan.

Reply:
This is a outdated comment.  Removed. Thanks.